### ISSUE TITLE
log-redaction: some data  is having ud tags with partial redaction level which is not supposed to have it

### ISSUE THREAD

**Issue by sridevi-15** [couchbase/sync_gateway#3533]
### Sync Gateway version
2.1.0-71

### Operating system
Centos

### Config file
```{
    "interface":":4984",
    "adminInterface": "0.0.0.0:4985",
    "maxIncomingConnections": 0,
    "maxCouchbaseConnections": 16,
    "maxFileDescriptors": 90000,
    "slowServerCallWarningThreshold": 500,
    "compressResponses": false,
    "log": ["*"],
    "logging":{
        "redaction_level": "partial"
    },
    "databases":{
        "db":{


            "num_index_replicas": 0,
            "num_index_replicas_housekeeping": 0,
            "enable_shared_bucket_access": true,
            "server":"http://192.168.33.10:8091",
            "bucket":"data-bucket",
            "password": "password"
        }
    }
}

```

### Log output
sg-info log : https://gist.github.com/sridevi-15/066131dba4a4564488eb91f57ac546d8

### Steps to reproduce
1.  Add sync-gateway config with redaction level partial
2.  start sync-gateway
3.  Create user and docs in sync_gaeway
4. Check the logs 

### Expected behavior
With partial redaction level, some data should be redacted like email, password, docs information, but it is also redacting db name, bucket name, api calls which is for 'complete' redaction level. I attached log file to review

### Actual behavior
I see that db, bucket, api calls are having ud tags with redaction level= partial 
Channels, public port, admin url are also having ud tags 

###Note:
Also see this log at the end of the log file
 #005: GET <ud>/</ud><ud> (as )</ud>
2018-04-28T17:11:56.908Z [INF] HTTP+: #005:     --> 200   (0.1 ms)

I do not see any data inside 'ud' tags .

**Comment by adamcfraser** [comment]
I agree - it looks like there is information being redacted here that shouldn't be.  In addition to the db and bucket names, my biggest concern is that we're redacting the full API path for HTTP logging.  That's not necessary, and has a significant impact on the usefulness of logs.

**Comment by bbrks** [comment]
Fixed incorrect tags and empty `(as )` in #3535 

I will address the full path HTTP redaction in #3537
